front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -165,6 +169,8 @@ const ScenarioV2 = () => {
     setCollapsedTimetable(isMacroMode);
   };
 
+  const handleNGEOperation = (op: NGEEvent) => ngeOperations(op, dispatch, timetableId!);
```

Comment:
I think this function should be moved in `NGE.tsx`. You can give a new props `timetableId` to `NGE`, and get the dispatch with `useAppDispatch` directly in `NGE.tsx`

Reply:
The motivation for this to be out of `NGE.tsx` is explained here: https://github.com/OpenRailAssociation/osrd/pull/8220#discussion_r1701793216